### PR TITLE
Bug 1747366: Allow use of global proxy

### PR DESCRIPTION
**Pull request by joelddiaz**
create a configmap that will be populated with the cluster's certificate bundle.
mark that configmap as an optional volume for the CCO pod.

TODO: watch the configmap for changes and restart COO as necessary.

### CODE REVIEW

File: manifests/01_deployment.yaml [hunk 1]
```diff
@@ -167,6 +169,9 @@ spec:
             cpu: 10m
             memory: 150Mi
         terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - name: cco-trusted-ca
+          mountPath: /etc/pki/ca-trust/extracted/pem/
```
Comment (dgoodwin) [inline manifests/01_deployment.yaml]: Does this behave ok when there's no proxy in play?

Comment (joelddiaz) [reply manifests/01_deployment.yaml]: supposedly. copied this from https://github.com/operator-framework/operator-marketplace/pull/245/files#diff-5e5f804a8ee2fb8c81711b728eb5a21dR76

Comment (joelddiaz) [reply manifests/01_deployment.yaml]: also why i pushed this PR. to see how CI handle this

**Comment by dgoodwin** [comment]
@joelddiaz will remove hold pending one more confirmation this is working.

**Pull request merged**